utils/protobuf-schema-utilities/src/main/java/io/apicurio/registry/utils/protobuf/schema/FileDescriptorUtils.java

```diff
@@ -400,6 +400,14 @@ private static MessageElement toMessage(FileDescriptorProto file, DescriptorProt
                     Collections.singletonList(reservedName));
             reserved.add(reservedElem);
         }
+        for (DescriptorProto.ReservedRange reservedRange : descriptor.getReservedRangeList()) {
+            List<Object> values = new ArrayList<>();
+            int start = reservedRange.getStart();
+            int end = reservedRange.getEnd() - 1;
```

Comment:
Good to add a comment explaining why the range is not end inclusive.

Reply:
There are no formal documentations in Protobuf that states the usage of open/closed range. The implementation is such though.

Reply:
Even if there's no formal documentation about this if there's anything that explains it I would like to have a comment explaining the reason.